### PR TITLE
Add Readmes to the Acorn files.

### PR DESCRIPTION
**Pull request by cloudnautique**

### CODE REVIEW

File: rds/aurora/mysql/cluster/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        WORDPRESS_DB_NAME: "instance"
+    }
+}
+```
+
+## Arguments
+
+| Name | Description | Type | Schema |
+|------|-------------|------|--------|
+| adminUsername | Name of the root/admin user. Default is admin. | string |
```
Comment (sosodev) [inline rds/aurora/mysql/cluster/README.md:36]: You've listed a `Schema` header but each row seems to only fill out the first three columns.

File: rds/aurora/mysql/cluster/README.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+| instanceClass | The instance class for the database server to use. Default is "burstable".  - burstable (good for dev/test and light workloads.) - burstableGraviton (good for dev/test and light workloads.) - memoryOptimized (good for memory intensive workloads.) **Updating this setting will cause downtime on the RDS instance.** | string |
+| instanceSize | The instance size to use.(medium, large, xlarge, or 2xlarge) Default is "medium". Not all instance sizes are available in all regions. **Updating this setting will cause downtime on the RDS instance.** | string |
+| parameters | RDS MySQL Database Parameters to apply to the cluster. Must be k/v string pairs(ex. max_connections: "1000"). | object |
+| skipSnapshotOnDelete | Do not take a final snapshot on delete or update and replace operations. Default is false. If skip is enabled the DB will be gone forever if deleted or replaced. | bool |
+| enablePerformanceInsights | Enable Performance insights. Default is false. | bool |
+| tags | Key value pairs of tags to apply to the RDS cluster and all other resources. | object |
+
+## Output Services
```
Comment (sosodev) [inline rds/aurora/mysql/cluster/README.md:47]: Is it user friendly to put the rendered Acornfile here or should we try to format this information better?

File: rds/aurora/mysql/serverless-v1/README.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
+# Aurora MySQL Serverless V1
+
+This Acorn creates an Aurora MySQL serverless cluster running on AWS RDS service. This Acorn is best used when you have a variable workload that sometimes is almost completely unused. If you have a stable workload, you should evaluate the cost of serverless vs. cluster based Aurora MySQL.
```
Comment (sosodev) [inline rds/aurora/mysql/serverless-v1/README.md:3]: Isn't serverless V2 preferred over V1? Perhaps we should let the user know that here.

**Review by sosodev — COMMENTED** [review]
A few notes

File: rds/aurora/mysql/cluster/README.md (hunk 1 above)
Comment (sosodev) [reply rds/aurora/mysql/cluster/README.md:36]: Oh and we should probably split the default value out into its own column.

File: rds/aurora/mysql/cluster/README.md (hunk 2 above)
Comment (cloudnautique) [reply rds/aurora/mysql/cluster/README.md:47]: I went with the Services, so this is what will be created by the service.sh script. These are the objects the user would interact with. Like `@{services.rds.ports.3306}` etc. Or data, there is no other place that data is really rendered.

**Review by sosodev — APPROVED** [review]

**Pull request merged**